docs/dev/api/README.md

```diff
@@ -4,6 +4,31 @@
 
 ## Modules
```

Comment:
maybe make this file binary, as it's autogenerated

Reply:
such that the diff are not given; maybe 'binary' is not the right term, it's the SVN term

Reply:
Ok I see, but the point is that we need to have this files committed on the release branch for the documentation, we could automatically commit on the release branch but we stay with this check and it is the responsability of the releaser to make doc update and pass the check.

Reply:
Yes, the file are in commit, it's just that the diff is not shown in PR (like, for eg zip files). But don't worry, forget this comment!